### DISCUSSION TITLE
2009/07/29/%E6%94%B6%E5%88%B0%E4%BA%86%E5%87%BA%E7%89%88%E7%A4%BE%E5%AF%84%E6%9D%A5%E7%9A%84%E6%B8%85%E6%A0%B7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#269] [Announcements]
# 2009/07/29/%E6%94%B6%E5%88%B0%E4%BA%86%E5%87%BA%E7%89%88%E7%A4%BE%E5%AF%84%E6%9D%A5%E7%9A%84%E6%B8%85%E6%A0%B7

这就说明编辑和排版都已结束，接下来等我再检查两遍就可以开始印刷了。

https://blog.qizhen.xyz/2009/07/29/%E6%94%B6%E5%88%B0%E4%BA%86%E5%87%BA%E7%89%88%E7%A4%BE%E5%AF%84%E6%9D%A5%E7%9A%84%E6%B8%85%E6%A0%B7

**Comment by ruanqizhen** [comment]
之前博客下的留言：

中梁
2009年09月6日 23:37 编辑
发现是北航出版社的，那离我很近，书一出我肯定立即过去买

回复
Qizhen
2009年09月2日 20:45 编辑
这本书还未上市。等有的卖了再通知大家。

回复
中梁
2009年09月2日 15:02 编辑
请问下怎么购买 我和labview 可以现在就可以订么

回复
Yigang
2009年08月22日 02:10 编辑
出版了立刻排队去买，然后到你家找你签名;-)

回复
Yu
2009年08月21日 09:10 编辑
恭喜哦：） 终于有一本可以强推荐的LabVIEW教材~~

回复
Wang
2009年07月29日 16:50 编辑
深圳南山区人们发来贺电

回复
Sierra
2009年07月29日 16:02 编辑
厉害的， 我3年前的稿子还沉睡在我的柜子里。

回复
l
2009年07月29日 15:53 编辑
呵呵，到时候一定要买一本拜读！

回复
帆
2009年07月29日 15:25 编辑
山东人民发来贺电！

回复
巍
2009年07月29日 15:18 编辑
望眼欲穿！好期待呀！

回复
